README.md

```diff
@@ -76,6 +76,36 @@ provisioner:
     organization: <ORGANIZATION_SLUG>
     privateKeyPath: ~/github.pem
 ```
+#### GitLab Runner
+
+To use the GitLab Runner provisioner you will need to [create a runner with an authentication token](https://docs.gitlab.com/ee/ci/runners/runners_scope.html#create-a-shared-runner-with-an-authentication-token).
+
+Minimal example:
+
+``` yml
+source: oci://ghcr.io/cirruslabs/macos-ventura-xcode:14.3.1
+provisioner:
+  type: gitlab
+  config:
+    gitlabURL: <GITLAB_INSTANCE_URL>
+    runnerToken: <RUNNER_TOKEN>
+```
+
+Full configuration:
+
+``` yml
+source: oci://ghcr.io/cirruslabs/macos-ventura-xcode:14.3.1
+provisioner:
+  type: gitlab
+  config:
+    gitlabURL: <GITLAB_INSTANCE_URL>
+    runnerToken: <RUNNER_TOKEN>
+    executor: <EXECUTOR> # defaults to 'shell'
+    maxNumberOfBuilds: <MAX_BUILDS> # defaults to '1'
+    downloadLatest: <DOWNLOAD_LATEST> # defaults to 'true'
+    downloadURL: <DOWNLOAD_URL> # defaults to GitLab official S3 bucket
+```
+

```

Comment:
Sorry it's not realistic for me to do this in the next week at least as I have some time off these days and am otherwise busy. Could we track this in a separate issue and resolve the convo in the meantime?